sharktuner/model_tuner/model_tuner.py

```diff
@@ -41,7 +42,7 @@ def read_flags_file(flags_file: str) -> list[str]:
         return file.read().splitlines()
 
 
-def arg_parse() -> argparse.Namespace:
+def arg_parse(parser_args: Sequence[str] | None = None) -> argparse.Namespace:
```

Comment:
I'd rather make it a sequence and disallow None -- we can modify users to pass in an empty list instead